Controllers/user.js

```diff
@@ -1,11 +1,12 @@
 let jwt = require("jsonwebtoken");
 var bcrypt = require("bcrypt");
 var ObjectId = require("mongodb").ObjectId;
-var handle = require("../Utils/error_handling");
-const { customValidationResult } = require("../Utils/error_handling");
+var handle = require("../utils/error_handling");
```

Comment:
I've sorta learned to think that "const" should be what's normally used when calling "require" or doing imports and stuff. I would suggest it but its likely that Im just being petty tho

Reply:
It is a good point to use const instead for these import statements